core/examples/luigi-sample-angular/src/app/services/helpers.ts

```diff
@@ -3,3 +3,7 @@ export const toTitleCase = (str: string = ''): string => {
     return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
   });
 };
+
+export const slugify = (str: string = ''): string => {
+  return str.toLowerCase().replace(/[^A-Z0-9]+/gi, '-');
```

Comment:
This will filter out all Umlaut-like characters which is not intended I guess.

Reply:
Intended, as it is used for pathSegment only in the demo app.

Reply:
Yes, sorry I misinterpreted the usage of this function.